src/App.js

```diff
@@ -1,9 +1,31 @@
-function App () {
+import React, { useState } from "react";
+import "./index.css";
+import Birds from "./Components/Birds.js";
+import Cart from "./Components/Cart";
+import Form from "./Components/Form";
+
+function App() {
+  const [cart, setCart] = useState([]);
+  const [form, formSubmit] = useState({
+    firstName: "",
+    lastName: "",
+    email: "",
+    zipCode: "",
+  });
+
```

Comment:
This code should be used inside `Form.js`. It controls and affects on ly the information inside the form so we don't need to lift it to App.js

Reply:
Thank you Raz. Got it!